hack/run-ci-e2e-test.sh

```diff
@@ -3,21 +3,54 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# TODO: Add input validation
+NODE_COUNT=""
+SKIP_NODE_DELETION=""
+KEY_PAIR_NAME=""
+
 WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 export CGO_ENABLED=0
 export CLUSTER_ADDR=$(oc cluster-info | head -n1 | sed 's/.*\/\/api.//g'| sed 's/:.*//g')
 
+while getopts ":n:k:s" opt; do
+  case ${opt} in
+    n ) # process option for providing existing VM credentials
+      NODE_COUNT=$OPTARG
+      ;;
+    k )
+      KEY_PAIR_NAME=$OPTARG
+      ;;
+    s ) # process option for skipping setup in VMs
+      SKIP_NODE_DELETION="-skip-node-deletion"
+      ;;
+    \? )
+      echo "Usage: $0 [-n] [-k] [-s]"
+      exit 0
+      ;;
+  esac
+done
+
 # Copy the cloud credentials and KUBESSH key path so that they can be used by operator
 cp $AWS_SHARED_CREDENTIALS_FILE /etc/cloud/credentials
 cp $KUBE_SSH_KEY_PATH /etc/private-key/private-key.pem
 
 # Get operator-sdk binary.
-# TODO: Make this to download the same version we have in go dependencies in gomod
-wget -O /tmp/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.15.2/operator-sdk-v0.15.2-x86_64-linux-gnu && chmod +x /tmp/operator-sdk
+# We don't care about the operator-sdk version we're using if running locally.
+if ! type operator-sdk > /dev/null; then
+  # TODO: Make this to download the same version we have in go dependencies in gomod
+  wget -O /tmp/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.15.2/operator-sdk-v0.15.2-x86_64-linux-gnu && chmod +x /tmp/operator-sdk
+  export PATH=$PATH:/tmp
```

Comment:
I'm not sure if this should be done, it could cause other side effects. Might be better to create an alias or something?

Reply:
I thought about this but I looked at the pod in CI which is controlled environment and it shouldn't cause an issue and in our machine when we are running it might be ok considering the `PATH` env variable is limited to scope of this bash script. However if you want this changed I can do it.